data-esnext.js

```diff
@@ -1064,6 +1110,7 @@ exports.tests = [
         firefox10: false,
         firefox60: false,
         chrome77: false,
+        rhino1_7: false,
```

Comment:
all these trailing commas in the data are going to break older browsers. please ensure none of them have trailing commas.

Reply:
Okeydoke

Reply:
I think these files are only consumed by `build.js` and aren't actually distributed via the web, but have made the change anyway

Reply:
Ah no there were a few trailing commas in tests (that weren't testing trailing comma support). Not added by me, but I've removed them too. I get the feeling that I could have kept all these commas in though.

Reply:
thanks! it's important the site keeps working in old browsers.